.github/workflows/unix.yml

```diff
@@ -85,7 +85,7 @@ jobs:
           - name: arm64
             runner: oqs-arm64
             container: openquantumsafe/ci-ubuntu-focal-arm64:latest
-            PYTEST_ARGS: --numprocesses=auto --maxprocesses=10 --ignore=tests/test_kat_all.py
+            PYTEST_ARGS: --maxprocesses=10 --ignore=tests/test_kat_all.py
```

Comment:
What does this have to do with Cross?

Reply:
My bad, it looks like this belongs to a commit from main unrelated to CROSS. I just rebased and force-pushed.